lmdeploy/pytorch/kernels/w8a8_triton_kernels.py

```diff
@@ -340,7 +311,7 @@ def _per_token_quant_int8(
     # Quant
     _absmax = tl.maximum(tl.max(tl.abs(y)), eps)
     y_s = _absmax / 127
-    y_q = tl.maximum(tl.minimum(tl.math.round(y / y_s), 127), -128).to(tl.int8)
+    y_q = tl.math.round(y / y_s).to(tl.int8)
```

Comment:
shall we add assertion here to make sure `tl.math.round(y / y_s)` is in [-128, 127]

Reply:
y_s = absmax(y) / 127. the value should be ok.
And it is not legal to add assert in triton kernel.